src/help/changelog.md

```diff
@@ -1,35 +1,37 @@
-#### December 7, 2016
-In this release we are making improvements to the Broker and responding to issues discovered through greater agency use.
+#### December 21, 2016
+On September 30, 2016, we released the full version of the DATA Act Broker that contained everything agencies need to test the data validation and submission process. Now we are making improvements to the Broker and responding to issues discovered through greater agency use.
 
-In this release there are bugfixes, improvements, and changes to the way the Data Broker handles certain conditions. 
+In this version of the Broker, users are able to generate D files outside of a submission, rules that were temporarily warnings are changed back to errors, the Broker is available at a new URL, rule B5 is updated, object class validations require a specific value when an object class is unknown, and MAX permissions allow users who are part of of multiple agency permission groups to have different permissions for different agencies.
 
-  - [Flex Fields](#/help?section=flexfields)
-  - [Comment box available for each file in certification](#/help?section=comments)
-  - [White space bugfix](#/help?section=whitespace)
-  - [Improvements to local install process](#/help?section=local)
-  - [Rounding error bugfix](#/help?section=rounding)
- 
-##### Flex Fields{section=flexfields}
+  - [Generate D files outside of a submission](#/help?section=Dfiles)
+  - [Temporary Warnings Changed Back to Errors](#/help?section=warnings2Errors)
+  - [New URL for the Data Broker](#/help?section=URL)
+  - [Update to Rule B5](#/help?section=B5)
+  - [Object Class Validation Update](#/help?section=objectClass)
+  - [MAX Group Permissions Allow for Different Permissions for a User Who is Part of a Multiple Agency Permission Group](#/help?section=maxGroup)
+  - [Browser Requirements & Known Issues](#/help?section=browser)
+  - [Accessibility Statement](#/help?section=accessibilityStatemen)
+
+##### Generate D Files Outside of a Submission{section=Dfiles}
 
- Users can now add additional columns to their submission files (A-C) that will be returned in their warning and error files. To use this feature, add any column to your submission data and prefix the header with "flex\_". For example, a column named "flex_reportingbureau" will be ignored for validation purposes but returned for any rows that have errors in the warnings and error reports. 
+Users will be able to generate D files outside the context of a submission. In other words, they won't need to validate A-C to be able to generate D files.
 
```

Comment:
We should add the link here to point agencies to the page where they can do this.